nucypher/cli/painting/help.py

```diff
@@ -17,15 +17,13 @@
 
 
 import click
-import maya
 from constant_sorrow.constants import NO_KEYSTORE_ATTACHED
 
 from nucypher.blockchain.eth.sol.__conf__ import SOLIDITY_COMPILER_VERSION
 from nucypher.characters.banners import NUCYPHER_BANNER
 from nucypher.config.constants import (
     DEFAULT_CONFIG_ROOT,
-    USER_LOG_DIR,
-    END_OF_POLICIES_PROBATIONARY_PERIOD
```

Comment:
Does this mean we no longer need this constant, nor need to update the date it ever again because tests would fail - 🥳 . Was it removed from the constants module?

Reply:
I believe it's still enforced by the other entry point (`Alice.grant`).  This removal just addresses the `alice` CLI entry point.